src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -41,14 +42,23 @@
 #include "utils/palloc.h"
 #include "utils/relcache.h"
 
+/* expression tree walker context for rewriting row references */
```

Comment:
On 2016-08-26 12:38:29 -0700, Jason Petersen wrote:
> > @@ -41,14 +42,23 @@
> >  #include "utils/palloc.h"
> >  #include "utils/relcache.h"
> >
> > +/* expression tree walker context for rewriting row references */
>
> Did we really? Where?

grep 'typedef struct' src/backend/distributed/

gives a bunch of hits.


> I'd actually love to go through and move all non-interface-related structs into implementation files, then…

Well, we should probably have drinks^Wa meeting and formalize some of
that. Next time when Marco, you and I are a tthe same place?